### PR TITLE
This patch implements upload-button in backupmode.

### PR DESCRIPTION
**Pull request by CGreweling**
It is possible to ingest the recorded file into mediapackage.
In Backup-mode the ca records simultaneously to the ca with the same name.
But it will not ingest the recording automatically.
The ingest can be triggered with the ui through clicking on a button.

### CODE REVIEW

**Comment by CGreweling** [comment]
@lkiesow have not recognized the conflicts here as i opened the pr. 
Solved them now. 
The 3.8 tests failed ``` Unprocessable Entity for url: https://coveralls.io/api/v1/jobs ```. I do not know what that means.

**Comment by lkiesow** [comment]
Rebased to fix build issue with coveralls

File: pyca/capture.py [hunk 1]
```diff
@@ -71,8 +71,14 @@ def start_capture(db, upcoming_event):
 
     # Set status
     # If part files exist, its an partial recording
-    p = any([glob.glob(f'{f}-part-*') for f in files])
-    state = Status.PARTIAL_RECORDING if p else Status.FINISHED_RECORDING
+    part_files = any([glob.glob(f'{f}-part-*') for f in files])
+    if part_files:
+        state = Status.PARTIAL_RECORDING
+    if config('agent')['backup_mode']:
```
Comment (lkiesow) [inline pyca/capture.py]: You always overwrite the status `PARTIAL_RECORDING`. That's probably not intentional.

```suggestion
    elif config('agent', 'backup_mode'):
```

File: pyca/db.py [hunk 2]
```diff
@@ -83,6 +83,7 @@ class Status(Constants):
     FAILED_UPLOADING = 6
     FINISHED_UPLOADING = 7
     PARTIAL_RECORDING = 8
+    FINISHED_RECORDING_PAUSED = 9
```
Comment (lkiesow) [inline pyca/db.py]: The status `finished recording paused` is somewhat confusing for users if you don't already know what it means. How can you pause a recording if it is already finished? What about using:

```suggestion
    PAUSED_AFTER_RECORDING = 9
```

**Review by lkiesow — CHANGES_REQUESTED** [review]
Generally looks good. I found one small error and have one suggestion for slightly different naming.

File: pyca/capture.py (hunk 1 above)
Comment (CGreweling) [reply pyca/capture.py]: Ah okay, i just copied that line from the old code, so the old code did not work anyway.^^

File: pyca/db.py (hunk 2 above)
Comment (CGreweling) [reply pyca/db.py]: I am fine with it, by the way that was your suggestion a while ago. ;)
But then this must also be changed i the ``` Event.vue ``` wil do that.

**Review by lkiesow — APPROVED** [review]

**Pull request merged**